### PR TITLE
Feel's Patch 

### PR DESCRIPTION
**Pull request by ghost**
Lots of grammar, spelling, consistency, and user changes.

### CODE REVIEW

File: files/guides/neu.md [hunk 1]
```diff
@@ -17,28 +17,25 @@ You can search for features using the scroll bar in the left part of this menu o
...
-> /neudevtest     
-
-##### Shows orb rage (exapmle: Mithril orb)
+##### Shows minion buff orb range 
 > /neuzeesphere     
 
-##### Shows any player profile (skycrypt but in game)
-> /neuprofile "player name"     
+##### Shows specified player's API
+> /pv "username"
```
Comment (koxx12-dev) [inline files/guides/neu.md:35]: why /pv ?

Comment (ghost) [reply files/guides/neu.md:35]: shorter than /neuprofile
and who uses /neuprofile

Comment (koxx12-dev) [reply files/guides/neu.md:35]: me and ok

Comment (ghost) [reply files/guides/neu.md:35]: weirdchamp

**Comment by ghost** [comment]
:malding: 

I fixed the grammar

**Pull request merged**